INSTRUCTIONS.md

```diff
@@ -1,627 +0,0 @@
-# Instructions
-
-If you'd like to follow along:
-
-1. `$ git clone https://github.com/kentcdodds/react-jest-workshop.git`
-2. `$ cd react-jest-workshop`
-4. `$ npm run setup`
-
-You'll notice that this repository is already (mostly) set up for a React project.
-It's a bit contrived and doesn't actually amount to anything but a couple
-disconnected components and a fairly worthless (non-flux) store.
-
-You'll also notice that right next to each module, there's a `.test.js` file where
-there's a single test that just verifies that your tests are running.
-
-We already have many of the same dependencies you would have in a normal react
-project like `babel` and friends, `react` (and friends), and
-`eslint`... and friends. However, to get our tests going, we're going to need
-a few more dependencies. Oh, and one more thing, let me introduce you to your new
-best friend:
-
-🐯 *- Hi! I'm Terry the Tiger! These instructions are really long and boring! So*
-*I'll pop up here and there where you'll be expected to actually do something!*
-*And if you really want to skip around, just copy me and <kbd>⌘</kbd>+<kbd>f</kbd>*
-*(or <kbd>CTRL</kbd>+<kbd>f</kbd> on windows) for me on the page. See you around!*
-
-# Starting here, things are old. I'm migrating this from AVA
-# If you wanna help, that'd be great
```

Comment:
Note this. I'd actually rather keep things around as a reference for when I finally do update the instructions. If you want to do that, be my guest! :D